jenkins-pipelines/rolling-upgrade-azure-image.jenkinsfile

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+rollingUpgradePipeline(
+    backend: 'azure',
+    azure_region_name: 'eastus',
+    azure_instance_type_db: 'Standard_L8s_v2',
```

Comment:
shouldn't these parameters go into a configuration file, other than hardcoded in the `jenkinsfile`?

Reply:
Fixed (moved most of the config items to a new config yaml file)